tests/assertions/internal-libraries.n

```diff
@@ -131,7 +131,7 @@ let pub main = [_: ()] -> cmd[()] {
 	let _ = times.sleep(1000)!
 	// Allow some wiggle room because timing is hard
 	// Also note that times.sleep takes ms but times.getTime outputs seconds
```

Comment:
Now that you've changed it, you probably can remove this comment now

Reply:
Done